packages/notifications/discord/src/index.ts

```diff
@@ -0,0 +1,59 @@
+import type { z } from "zod";
+
+import type {
+  basicMonitorSchema,
+  selectNotificationSchema,
+} from "@openstatus/db/src/schema";
+
+const postToWebhook = async (content: string, webhookUrl: string) => {
+  await fetch(webhookUrl, {
+    method: "POST",
+    headers: {
+      "Content-Type": "application/json",
+    },
+    body: JSON.stringify({
+      content,
+      avatar_url:
+        "https://img.stackshare.io/service/104872/default_dc6948366d9bae553adbb8f51252eafbc5d2043a.jpg",
+      username: "OpenStatus Notifications",
+    }),
+  });
+};
+
+export const sendDiscordMessage = async ({
+  monitor,
+  notification,
+}: {
+  monitor: z.infer<typeof basicMonitorSchema>;
+  notification: z.infer<typeof selectNotificationSchema>;
+}) => {
+  const notificationData = JSON.parse(notification.data);
+  const { discord: webhookUrl } = notificationData; // webhook url
+  const { name } = monitor;
+
+  try {
+    await postToWebhook(
+      `Your monitor ${name} is down 🚨
+
+      Your monitor with url ${monitor.url} is down.`,
```

Comment:
Can we have markdown here, like `[monitor.url](https://openstatus.dev/app/...)` to add the path to the monitor. We might need the `workspaceSlug` here. Could also be a thing for "email" (not the md but the link).